### ISSUE TITLE
Cannot view XForm videos on mobile device when offline

### ISSUE THREAD

**Issue by abbyad** [medic/medic#1926]
Follow up to #1157, we want videos to be seen on mobile devices when offline.

For now this is not a hard requirement for planned deployments, but feature would be used when it becomes available.

**Comment by garethbowen** [comment]
In testing this I found an unrelated bug introduced in a bad code merge here: https://github.com/medic/medic/commit/b5ca39131f7011b385213f07900ad35d46bb3474#diff-196cac173cf74d843fcdc5e9ed5ff7cdR72

I'm fixing it as part of this issue so both can be tested.

However, once that bug is out of the way I had no trouble loading video attachments while offline. To test this I used the [multimedia](https://github.com/medic/medic/blob/master/demo-forms/multimedia.xml) form with a video and image from https://sample-videos.com . Then I synced as an offline user, turned to offline mode in dev tools, refreshed the page, loaded the form, and played the video.
